lib/shared/bucketing-assembly-script/assembly/managers/clientCustomDataManager.ts

```diff
@@ -1,16 +1,27 @@
+import { CustomDataValue } from '../types'
 import { JSON } from 'assemblyscript-json/assembly'
+import { getJSONObjFromCustomDataMap } from '../helpers/jsonHelpers'
 
-const _clientCustomData: Map<string, JSON.Obj> = new Map()
+const _clientCustomData: Map<string, Map<string, CustomDataValue>> = new Map()
```

Comment:
I'm storing the client custom data in both Map and JSON form. See the comment in eventQueueManager below for an explanation.